src/parse.coffee

```diff
@@ -14,9 +13,11 @@ createWarning = (message) ->
 
 parse = (source, callback) ->
   warning = null
-  mediaType = deckardcain.identify(source)
+  adapters = fury.detect(source)
 
-  unless mediaType
+  if adapters.length
+    mediaType = adapters[0].mediaTypes[0]
```

Comment:
Which adapter will be used by Fury to parse the description? The first one? The main point of the code is to use API Blueprint as a fallback format (i.e. in case Fury / Deckard are not able to detect the format) and to pass the media type further.